app/assets/javascripts/components/markdown-editor.js

```diff
@@ -68,6 +81,16 @@ MarkdownEditor.prototype.toggle = function (element) {
   }
 }
 
+MarkdownEditor.prototype.setContextualGuidancePosition = function () {
+  var guidanceId = this.$input.getAttribute('data-contextual-guidance')
+  if (guidanceId) {
+    var $guidance = document.querySelector('#' + guidanceId + ' .app-c-contextual-guidance')
```

Comment:
How come this is overriding the style 'position: absolute' - could we not just change the style?

Reply:
We're overwriting the position only for the contextual guidance that relates to the markdown editor as it contains the expandable toolbar which can overlap the footer.